srfi-119/README

```diff
@@ -1,3 +1,7 @@
+# SPDX-FileCopyrightText: 2014 Arne Babenhauserheide <arne_bab@web.de>
+#
+# SPDX-License-Identifier: MIT
+
```

Comment:
Putting this at the top of the README looks strange. Can it be at the bottom?

Reply:
It can't be at the bottom, but I'll look into what comment I should use for markdown, thanks for the feedback!

Reply:
Done via `reuse annotate -l MIT -s html -y 2014 -c 'Arne Babenhauserheide <arne_bab@web.de>' srfi-119/README`; html is the same style as used for markdown in reuse.